docs/using.rst

```diff
@@ -296,7 +297,24 @@ need to issue this command in normal circumstances.
 
 ``--expand`` tells Certbot to update an existing certificate with a new
 certificate that contains all of the old domains and one or more additional
-new domains.
+new domains. With the ``--expand`` option, use the ``-d`` option to specify
+all existing domains and one or more new domains.
+
+Example:
+
+.. code-block:: none
+
+  certbot --expand -d existing.com -d example.com,newdomain.com
```

Comment:
Question: I think the original purpose of these two `--expand` examples was to show that you can use `-d <domain 1>,<domain 2>,<domain 3>` or `-d <domain 1> -d <domain 2> -d <domain 3>`. Now the first example uses both versions (which is also valid). Is this what you wanted?